doc/source/conf.py

```diff
@@ -0,0 +1,75 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# For the full list of built-in configuration values, see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+import datetime
+import os
+import sys
+
+sys.path.insert(0, os.path.abspath("../../src"))
+
+# -- Project information -----------------------------------------------------
+
+project = "lammmpsio"
+year = datetime.date.today().year
+copyright = f"2021-{year}, Auburn University"
+author = "Michael P. Howard"
+version = "0.2.1"
+release = "0.2.1"
```

Comment:
Replace with correct version number. We will also want to add bumpversion support to this project so we can keep the version up-to-date (see relentless).